docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -504,6 +510,95 @@ The build process for docs.ansible.com will know where to find the module docs.
   * ReStructured Text docs (anything under ``docs/docsite/rst/``)
   * Files that never existed in ``ansible/ansible:devel``
 
+.. _testing_collections:
+
+Testing Collections
+===================
+
+The main tool for testing collections is ``ansible-test``, Ansible's testing tool described in :ref:`developing_testing`. This tool provides several compile and sanity checks, and allows to run unit and integration tests for plugins.
+
+``ansible-test`` must always be executed in the root directory of a collection. In the following, we will describe how to run ``ansible-test`` with Docker containers. This is the easiest way to use ``ansible-test`` as no special requirements have to be installed, and this is also the way it is run in Shippable both in Ansible's main repository and in the large community collections such as `community.general <https://github.com/ansible-collections/community.general/>`_ and `community.network <https://github.com/ansible-collections/community.network/>`_.
+
+Compile and Sanity Tests
+------------------------
+
+To run all compile and sanity tests, execute::
+
+    ansible-test sanity --docker default --color -v
```

Comment:
I'd omit the `--color` option, as it's normally auto-detected for interactive use. It should only be needed when piping stdout or on CI systems which do not report stdout as a TTY.

```suggestion
    ansible-test sanity --docker default -v
```

The same applies to the other examples below.

I'd also consider using `--docker` instead of `--docker default` (but with a note mentioning that using a non-option after it will be seen as a container name).

Reply:
I'll remove `--color` then.

I included `default` after `--docker` to avoid that when people remove `--color -v`, it suddenly reports "strange" errors when they list an explicit target.